.github/workflows/functional-test.yml

```diff
@@ -93,14 +89,15 @@ jobs:
         cwd=$(pwd)
         pushd "$cwd"
         cd ~
-        CHROMEDRIVER_VERSION="${{ matrix.chromedriverVersion }}" appium driver install --source=local "$cwd"
+        appium driver install --source=local "$cwd"
         appium driver doctor uiautomator2
         nohup appium server \
           --port=$APPIUM_TEST_SERVER_PORT \
           --address=$APPIUM_TEST_SERVER_HOST \
           --relaxed-security \
           --log-no-colors \
           --log-timestamp \
+          --allow-insecure chromedriver_autodownload \
```

Comment:
I think this is not necessary since we already set `--relaxed-security ` which includes all security flags stuff

Reply:
yes, makes sense